integration/sawtooth_integration/docker/test_poet_liveness.yaml

```diff
@@ -138,8 +142,12 @@ services:
       PYTHONPATH: "/project/sawtooth-core/consensus/poet/common:\
         /project/sawtooth-core/consensus/poet/simulator:\
         /project/sawtooth-core/consensus/poet/core:\
+<<<<<<< HEAD
         /project/sawtooth-core/sdk/python:\
         /project/sawtooth-core/families/block_info"
+=======
+        /project/sawtooth-core/families/block_info/python"
+>>>>>>> 1c80e9a2... Add block-info tp in rust
```

Comment:
Cleanup git cruft from rebase

Reply:
Hi Adam, are you looking at the right version of the PR? For me this shows as outdated.   I already removed the block_info.rs and cleaned this up in my last commit. Though, I did not push the fix to the copyrights headers. I'll do that now.

Reply:
Adam looked at each commit individual. This change should be squashed into the commit that introduced the issue.